dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/heimathafen/HeimathafenDataController.kt

```diff
@@ -26,11 +27,13 @@ class HeimathafenDataController(
     @Autowired var myDataManager: DataManager,
     @Autowired var myMetaDataManager: DataMetaInformationManager,
     @Autowired var myObjectMapper: ObjectMapper,
+    @Autowired var myPermissionChecker: PermissionChecker,
```

Comment:
Ist das Autowired noch hier drin, wiel die Toolbox noch nicht wieder gelaufen ist?

Reply:
Not relevant